.github/workflows/ci.yml

```diff
@@ -36,6 +36,7 @@ jobs:
         rust_target:
           [
             x86_64-unknown-linux-musl,
+            aarch64-unknown-linux-musl,
             armv7-unknown-linux-gnueabihf,
             x86_64-apple-darwin,
           ]
```

Comment:
(I know you didn't start this but,) can we use a bulleted yaml list here since it is spanning more than one line? E.g.

```yaml
rust_target:
  - x86_64-unknown-linux-musl
  - aarch64-unknown-linux-musl
  - armv7-unknown-linux-gnueabihf
  - x86_64-apple-darwin
```

Reply:
updated